<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSoundKnete</class>
 <widget class="QMainWindow" name="DSoundKnete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DSoundKnete</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="VideoWidget" name="inputFrameTop">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VideoWidget" name="processedFrameTop">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>150</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VideoWidget" name="inputFrameFront">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VideoWidget" name="processedFrameFront">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>150</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="hmax">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="smin">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="hmin">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="smax">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vmin">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vmax">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="hminLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>255</string>
    </property>
   </widget>
   <widget class="QLabel" name="hmaxLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="sminLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="smaxLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>255</string>
    </property>
   </widget>
   <widget class="QLabel" name="vminLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>320</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="vmaxLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>340</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>255</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Video</string>
    </property>
    <widget class="QMenu" name="menuVideodatei_ffnen">
     <property name="title">
      <string>Videodatei Öffnen ...</string>
     </property>
     <addaction name="actionVideo_Top"/>
     <addaction name="actionVideo_Front"/>
    </widget>
    <addaction name="menuVideodatei_ffnen"/>
    <addaction name="actionKamera_ffnen"/>
   </widget>
   <widget class="QMenu" name="menuPlay">
    <property name="title">
     <string>Play</string>
    </property>
    <addaction name="actionPlay"/>
   </widget>
   <addaction name="menuVideo"/>
   <addaction name="menuPlay"/>
  </widget>
  <action name="actionKamera_ffnen">
   <property name="text">
    <string>Kamera Öffnen</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionVideo_Top">
   <property name="text">
    <string>Video Top</string>
   </property>
  </action>
  <action name="actionVideo_Front">
   <property name="text">
    <string>Video Front</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QLabel</extends>
   <header>videowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>hmax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hmax</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smax</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vmax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vmax</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vmin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vmin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hmin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hminLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hmax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hmaxLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smaxLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sminLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vmin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vminLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vmax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vmaxLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
